docs/book/src/developer/testing.md

```diff
@@ -53,36 +53,100 @@ The following guidelines should be followed when developing E2E tests:
 
 See [e2e development] for more information on developing e2e tests for CAPI and external providers.
 
-## Running the end-to-end tests
+## Running the end-to-end tests locally
 
-`make docker-build-e2e` will build the images for all providers that will be needed for the e2e test.
+Usually, the e2e tests are executed by Prow either pre-submit (on PRs) or periodically on certain branches
+(e.g. the default branch). Those jobs are defined in the kubernetes/test-infra repository in [config/jobs/kubernetes-sigs/cluster-api](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes-sigs/cluster-api).
+For development and debugging those tests can also be executed locally.
+
+### Prerequisites
+
+`make docker-build-e2e` will build the images for all providers that will be needed for the e2e tests.
+
+### Test execution via ci-e2e.sh
+
+To run the tests locally via command line we can just take a look at the Job configuration and execute the same locally.
```

Comment:
since we are instructing users on how they can any test for themselves, maybe make this next sentence a little more direct. 
```suggestion
To run a test locally via the command line, you should look at the Prow Job configuration for the test you want to run and then execute the same container commands locally.
```